src/main/java/com/jw/resources/FinalizeOrderResource.java

```diff
@@ -13,6 +13,8 @@
 @RequiredArgsConstructor
 @ApplicationScoped
 @Slf4j
+
+//REVIEW-VINI: minor - Maybe we can move this method to OrderResource?
```

Comment:
Reason why I created new classes (Resource and Service) for finalizing order, was willingness to separate it from OrderResource which is responsible for basic CRUD operations on order. I thought new business logic may require new class, but maybe I'm overthinking it